docs/storage/prometheus.md

```diff
@@ -14,79 +14,79 @@ To monitor cAdvisor with Prometheus, simply configure one or more jobs in Promet
-`container_memory_bandwidth_local_bytes` | Gauge | Local memory bandwidth usage statistics for container counted with RDT Memory Bandwidth Monitoring (MBM). | bytes
-`container_memory_cache` | Gauge | Total page cache memory | bytes
-`container_memory_failcnt` | Counter | Number of memory usage hits limits |
-`container_memory_failures_total` | Counter | Cumulative count of memory allocation failures |
-`container_memory_max_usage_bytes` | Gauge | Maximum memory usage recorded | bytes
-`container_memory_rss` | Gauge | Size of RSS | bytes
-`container_memory_swap` | Gauge | Container swap usage | bytes
-`container_memory_mapped_file` | Gauge | Size of memory mapped files | bytes
-`container_memory_usage_bytes` | Gauge | Current memory usage, including all memory regardless of when it was accessed | bytes
-`container_memory_working_set_bytes` | Gauge | Current working set | bytes
-`container_network_receive_bytes_total` | Counter | Cumulative count of bytes received | bytes
-`container_network_receive_packets_dropped_total` | Counter | Cumulative count of packets dropped while receiving |
-`container_network_receive_packets_total` | Counter | Cumulative count of packets received |
-`container_network_receive_errors_total` | Counter | Cumulative count of errors encountered while receiving |
-`container_network_transmit_bytes_total` | Counter | Cumulative count of bytes transmitted | bytes
-`container_network_transmit_packets_total` | Counter | Cumulative count of packets transmitted |
-`container_network_transmit_packets_dropped_total` | Counter | Cumulative count of packets dropped while transmitting |
-`container_network_transmit_errors_total` | Counter | Cumulative count of errors encountered while transmitting |
-`container_network_tcp_usage_total` | Gauge | tcp connection usage statistic for container |
-`container_network_tcp6_usage_total` | Gauge | tcp6 connection usage statistic for container |
-`container_network_udp_usage_total` | Gauge | udp connection usage statistic for container |
-`container_network_udp6_usage_total` | Gauge | udp6 connection usage statistic for container |
-`container_processes` | Gauge | Number of processes running inside the container |
-`container_referenced_bytes` | Gauge |  Container referenced bytes during last measurements cycle based on Referenced field in /proc/smaps file, with /proc/PIDs/clear_refs set to 1 after defined number of cycles configured through `referenced_reset_interval` cAdvisor parameter.</br>Warning: this is intrusive collection because can influence kernel page reclaim policy and add latency. Refer to https://github.com/brendangregg/wss#wsspl-referenced-page-flag for more details. | bytes
+`container_llc_occupancy_bytes` | Gauge | Last level cache usage statistics for container counted with RDT Memory Bandwidth Monitoring (MBM). | bytes | resctrl
```

Comment:
resctrl option is not visible in cAdvisor help

Reply:
Now it is.